src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/ApplicationsController.kt

```diff
@@ -407,31 +404,20 @@ class ApplicationsController(
   override fun applicationsApplicationIdDocumentsGet(applicationId: UUID): ResponseEntity<List<Document>> {
     val deliusPrincipal = httpAuthService.getDeliusPrincipalOrThrow()
     val username = deliusPrincipal.name
-
     val application = extractEntityFromCasResult(applicationService.getApplicationForUsername(applicationId, username))
 
-    val documentsFromApDeliusFlag = featureFlagService.getBooleanFlag("get-documents-from-ap-delius")
-    val documents = when (val documentsResult = getDocuments(isDocumentFromAPDelius = documentsFromApDeliusFlag, application.crn)) {
-      is AuthorisableActionResult.NotFound -> throw NotFoundProblem(application.crn, "Person")
-      is AuthorisableActionResult.Unauthorised -> throw ForbiddenProblem()
-      is AuthorisableActionResult.Success -> documentsResult.entity
-    }
-
-    val transformedDocuments = when (documents) {
-      is GroupedDocuments -> documentTransformer.transformToApi(documents)
```

Comment:
I think this function on DocumentTransformer can now be removed as the only thing that should be using it is unit tests:

`fun transformToApi(groupedDocuments: GroupedDocuments)`